tests/DropboxTest.php

```diff
@@ -1,10 +1,92 @@
 <?php
-use PHPUnit\Framework\TestCase;
 
+namespace Kunnu\Dropbox\Tests;
+
+use GuzzleHttp\Psr7\Request;
+use Kunnu\Dropbox\Dropbox;
+use Kunnu\Dropbox\Models\FileMetadata;
+use Kunnu\Dropbox\Models\FolderMetadata;
+
+/**
+ * @author Cristiano Cinotti <cristianocinotti@gmail.com>
+ */
 class DropboxTest extends TestCase
 {
-    public function testDemo()
+    public function testGetMetadataOnFile()
+    {
+        $responseBody = <<<JSON
+{
+    ".tag": "file",
+    "name": "fake_file.txt",
+    "path_lower": "/fake_dir/fake_file.txt",
+    "path_display": "/fake_dir/fake_file.txt",
+    "id": "id:a4ayc_80_OEAAAAAAAAAXw",
+    "client_modified": "2015-05-12T15:50:38Z",
+    "server_modified": "2015-05-12T15:50:38Z",
+    "rev": "a1c10ce0dd78",
+    "size": 7212,
+    "content_hash": "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855"
+}           
+JSON;
+        $dropbox = $this->getDropbox($responseBody);
+
+        $metadata = $dropbox->getMetadata('/fake_dir/fake_file.txt');
+
+        //Test request
+        $this->assertCount(1, $this->getHistory(), 'One call to Dropbox api');
+
+        /** @var Request $request */
+        $request = $this->getHistory()[0]['request'];
+        $this->assertEquals('POST', $request->getMethod(), 'Submit Dropbox call via POST method');
+        $this->assertEquals('https://api.dropboxapi.com/2/files/get_metadata', $request->getUri(), 'Correct uri');
+        $this->assertEquals('{"path":"\/fake_dir\/fake_file.txt"}', $request->getBody()->getContents(), 'Correct request body');
+        
+        //Test response
+        $this->assertInstanceOf(FileMetadata::class, $metadata);
+        $this->assertEquals('fake_file.txt', $metadata->getName());
+        $this->assertEquals('/fake_dir/fake_file.txt', $metadata->getPathLower());
+        $this->assertEquals('id:a4ayc_80_OEAAAAAAAAAXw', $metadata->getId());
+    }
+
+    public function testGettMetadataOnFolder()
```

Comment:
There's a tiny typo: `testGettMetadataOnFolder` must be `testGetMetadataOnFolder`.